Ports/cavestory/package.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+port="cavestory"
+version="2.6.5-1"
+files="https://github.com/nxengine/nxengine-evo/archive/refs/heads/master.zip master.zip d013acbf399938b5d3d10fd4d37817b630559e2b24b90576cde6f88d445ea150"
```

Comment:
Please pin this to a specific revision or tag instead of `master`. Otherwise, the hash will no longer match as soon as the repository is updated.